spikeinterface/postprocessing/unit_localization.py

```diff
@@ -214,7 +218,7 @@ def estimate_distance_error_with_log(vec, wf_ptp, local_contact_locations, maxpt
 
 
 def compute_monopolar_triangulation(waveform_extractor, optimizer='minimize_with_log_penality',
-                                    radius_um=50, max_distance_um=1000, return_alpha=False):
+                                    local_radius_um=75, max_distance_um=1000, return_alpha=False, enforce_decrease=False):
```

Comment:
I would keep `radius_um`, since it's used in several instances (e.g. also for sparsity, plotting, etc..)

Reply:
Back to radisu_um in all methods at the unit level